pkg/apis/numaflow/v1alpha1/pipeline_types.go

```diff
@@ -205,6 +205,10 @@ func (p Pipeline) GetSideInputsStoreName() string {
 	return fmt.Sprintf("%s-%s", p.Namespace, p.Name)
 }
 
+func (p Pipeline) GetServingSourceStreamName() string {
```

Comment:
We will not allow multiple Serving sources in one pipeline, correct?

Reply:
there is nothing stopping us from doing so.

Reply:
Then the stream name should not be tied to pipeline name only.

Reply:
yes, I will tackle it in a separate PR. It needs more changes.